src/ui/profile/AddProfile.js

```diff
@@ -0,0 +1,146 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import styles from '../styles/profile.module.css';
+import ProfileModel from '../../models/profile';
+import type from '../../utils/types';
+
+const BEYOND_ID_LENGTH = 8;
+
+function AddProfileForms(props) {
+  const { onAddProfile, onCancel, onError } = props;
```

Comment:
Another place you can destructure the props in argument position

Reply:
I figured this out eventually. I've been refactoring them as I've found them